charts/ndm/values.yaml

```
@@ -0,0 +1,87 @@
+# Default values for ndm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+release:
+  version: "0.8.0"
+ndm:
+  enabled: true
+  image:
+    # Make sure that registry name end with a '/'.
+    # For example : quay.io/ is a correct value here and quay.io is incorrect
+    registry:
+    repository: openebs/node-disk-manager-amd64
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: 0.8.0
+  sparse:
+    path: "/var/openebs/sparse"
+    size: "10737418240"
+    count: "0"
+  nodeSelector: {}
+  tolerations: []
+  filters:
+    enableOsDiskExcludeFilter: true
+    enableVendorFilter: true
+    excludeVendors: "CLOUDBYT,OpenEBS"
+    enablePathFilter: true
+    includePaths: ""
+    excludePaths: "loop,fd0,sr0,/dev/ram,/dev/dm-,/dev/md,/dev/rbd"
+  probes:
+    enableSeachest: false
+    enableUdevProbe: false
+    enableSmartProbe: false
+  healthCheck:
+    initialDelaySeconds: 30
+    periodSeconds: 60
+
+ndmOperator:
+  enabled: true
+  image:
+    registry:
+    repository: openebs/node-disk-operator-amd64
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: 0.8.0
+  nodeSelector: {}
+  tolerations: []
+  healthCheck:
+    initialDelaySeconds: 30
+    periodSeconds: 60
+  readinessCheck:
+    initialDelaySeconds: 4
+    periodSeconds: 10
+    failureThreshold: 1
+  replicas: 1
+  upgradeStrategy: Recreate
+
+cleanupUtil:
+  image:
+    registry:
+    repository: openebs/linux-utils
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: 2.0.0
+
+crd:
+  enableInstall: true
+
+featureGates:
+  enabled: false
```

featureGates should also be enabled.
```suggestion
  enabled: true
```